src/nanosvgrast.h

```diff
@@ -956,7 +956,7 @@ static float nsvg__clampf(float a, float mn, float mx) { return a < mn ? mn : (a
 
 static unsigned int nsvg__RGBA(unsigned char r, unsigned char g, unsigned char b, unsigned char a)
 {
-	return (r) | (g << 8) | (b << 16) | (a << 24);
+	return (r) | (g << 8) | (b << 16) | ((unsigned int)a << 24);
```

Comment:
Let's cast all `r,g,b,a` to `unsigned int` here so that it's obvious what happens. I dont mind it being verbose.